float32 x
float32 y
float32 z
int32 leg_no
---
float32 theta1
float32 theta2
float32 theta3
bool success
